packages/tar-eio/tar-eio.2.6.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Decode and encode tar format files using Eio"
+description: """
+tar is a library to read and write tar files with an emphasis on
+streaming.  This library uses Eio to provide a portable tar library.
+"""
+maintainer: ["Reynir Björnsson <reynir@reynir.dk>" "dave@recoil.org"]
+authors: ["Dave Scott" "Thomas Gazagnaire" "David Allsopp" "Antonin Décimo"]
+license: "ISC"
+tags: ["org:xapi-project" "org:mirage"]
+homepage: "https://github.com/mirage/ocaml-tar"
+doc: "https://mirage.github.io/ocaml-tar/"
+bug-reports: "https://github.com/mirage/ocaml-tar/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08.0"}
+  "eio" {>= "0.10.0" & < "0.12"}
```

Comment:
Is it already known to break for eio 0.12 or newer?

Reply:
seems like it:

```
       > File "eio/tar_eio.mli", line 23, characters 57-72:
       > 23 | val get_next_header : ?level:Tar.Header.compatibility -> Eio.Flow.source ->
       >                                                               ^^^^^^^^^^^^^^^
       > Error: The type constructor Eio.Flow.source expects 1 argument(s),
       >        but is here applied to 0 argument(s)
```